save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/WarnPlugin.kt

```diff
@@ -75,14 +86,20 @@ class WarnPlugin(
         generalConfig: GeneralConfig?): List<TestResult> {
         val expectedWarnings: WarningMap = mutableMapOf()
         paths.forEach { path ->
+            var i = 1
+            val linesFile = fs.readLines(path)
             expectedWarnings.putAll(
                 fs.readLines(path)
```

Comment:
```suggestion
                linesFile
```
Why read file twice?